src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesLauncher.java

```diff
@@ -87,6 +87,7 @@ public KubernetesLauncher() {
     }
 
     @Override
+    @SuppressFBWarnings(value = "IS2_INCONSISTENT_SYNC", justification = "Only this accessor is called without synchronized.")
     public boolean isLaunchSupported() {
```

Comment:
https://github.com/jenkinsci/kubernetes-plugin/commit/3e0fd5045caa97962ac499e8f0a87776a26ff0c8

Better fix it with making this method `synchronized` as well.

Reply:
Done.